### ISSUE TITLE
Can only install one of: kriswallsmith/buzz dev-master, kriswallsmith/buzz v0.7

### ISSUE THREAD

**Issue by kayue** [widop/WidopGoogleAnalyticsBundle#4]
Problem 1
    - Can only install one of: kriswallsmith/buzz dev-master, kriswallsmith/buzz v0.7.
    - Can only install one of: kriswallsmith/buzz v0.7, kriswallsmith/buzz dev-master.
    - widop/http-adapter-bundle dev-master requires kriswallsmith/buzz dev-master -> satisfiable by kriswallsmith/buzz dev-master.
    - widop/google-analytics-bundle dev-master requires widop/http-adapter-bundle dev-master -> satisfiable by widop/http-adapter-bundle dev-master.
    - Installation request for widop/google-analytics-bundle dev-master -> satisfiable by widop/google-analytics-bundle dev-master.
    - Installation request for kriswallsmith/buzz == 0.7.0.0 -> satisfiable by kriswallsmith/buzz v0.7.

**Comment by kayue** [comment]
I managed to install the bundle through the following settings:

```
    "widop/google-analytics-bundle": "dev-master",
    "kriswallsmith/buzz": "dev-master as v0.7"
```

**Comment by clemherreman** [comment]
Can you give your `composer.json` file? I think this is because of a too high `minimum_stability` you've set, as the `WidopGoogleAnalyticBundle` and one of its dependencies, `WidopHttpAdapterBundle` are using `dev-master` version of `Buzz`, where your `composer.json` need _stable_ packages.

ping @egeloen

**Comment by kayue** [comment]
You might be right. I am using Symfony 2.2.

```
{
    "name": "symfony/framework-standard-edition",
    "description": "The \"Symfony Standard Edition\" distribution",
    "autoload": {
        "psr-0": { "": "src/" }
    },
    "require": {
        "php": ">=5.3.3",
        "symfony/symfony": "2.2.*",
        "doctrine/orm": ">=2.2.3,<2.4-dev",
        "doctrine/doctrine-bundle": "1.0.*",
        "twig/extensions": "1.0.*",
        "symfony/assetic-bundle": "2.1.*",
        "symfony/swiftmailer-bundle": "2.1.*",
        "symfony/monolog-bundle": "2.1.*",
        "sensio/distribution-bundle": "2.2.*",
        "sensio/framework-extra-bundle": "2.2.*",
        "sensio/generator-bundle": "2.2.*",
        "jms/security-extra-bundle": "1.4.*",
        "jms/di-extra-bundle": "1.3.*",

        "excelwebzone/omlex-bundle": "dev-master",
        "excelwebzone/recaptcha-bundle": "dev-master",
        "knplabs/knp-disqus-bundle": "dev-master",
        "mopa/bootstrap-bundle": "dev-master",
        "predis/predis": "0.8.*",
        "snc/redis-bundle": "dev-master",
        "stof/doctrine-extensions-bundle": "dev-master",
        "twitter/bootstrap": "dev-master",
        "knplabs/knp-paginator-bundle": "dev-master",
        "hearsay/require-js-bundle": "dev-master",
        "kachkaev/assets-version-bundle": "dev-master",
        "avalanche123/imagine-bundle": "dev-master",
        "eko/feedbundle": "dev-master",
        "widop/google-analytics-bundle": "dev-master",
        "kriswallsmith/buzz": "dev-master as v0.7"
    },
    "scripts": {
        "post-install-cmd": [
            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile"
        ],
        "post-update-cmd": [
            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile"
        ]
    },
    "config": {
        "bin-dir": "bin"
    },
    "minimum-stability": "dev",
    "extra": {
        "symfony-app-dir": "app",
        "symfony-web-dir": "web"
    },
    "repositories": [
        {
            "type": "package",
            "package": {
                "name": "twitter/bootstrap",
                "version": "master",
                "dist": {
                    "type": "zip",
                    "url": "https://github.com/twitter/bootstrap/zipball/master"
                }
            }
        },
        {
            "type": "vcs",
            "url": "http://github.com/101medialab/KnpDisqusBundle"
        },
        {
            "type": "vcs",
            "url": "http://github.com/101medialab/FeedBundle"
        }
    ]
}
```

**Comment by egeloen** [comment]
All will be fixed when #9 will be merged
